src/GameMain/ScoreBoard.java

```diff
@@ -68,9 +68,93 @@ public ScoreBoard(GameView theView) {
         restartButton.addActionListener(this);
     }
 
+    @Override
+    public void paintComponent(Graphics g) {
```

Comment:
The paintComponent method is doing a lot more than just painting. It's also handling data loading and calculations (loadScore()). Try to separate concerns, keeping methods focused on specific tasks for better code organization and understanding.